src/main/c/truffleposix/truffleposix.c

```diff
@@ -467,7 +467,7 @@ static void copy_stat(struct stat *native_stat, struct truffleposix_stat* buffer
 #elif defined(HAVE_STRUCT_STAT_ST_ATIMENSEC)
   buffer->atime_nsec = (long)native_stat->st_atimensec;
 #else
-  buffer->atime_nsec = 0
+  buffer->atime_nsec = 0;
```

Comment:
@lewurm This is the difference with your branch, would be good to pick it there

Reply:
I saw the syntax error from this in practice on my machine.